ideals.md

```diff
@@ -10,75 +10,52 @@ These ideals also help define the scope of Cockpit. That scope is limited, and n
 
 ### Cockpit is discoverable
 
-It should be possible to discover how to perform tasks through Cockpit without reading a help file. Astute readers may note that we have not completely acheived this yet, but this drives design decisions for Cockpit.
+It should be possible to discover how to perform tasks through Cockpit without reading a help file.
 
-### Cockpit is remotable
+### Cockpit directly interacts with and reacts to the system
 
-Cockpit can be effectively used when you are not sitting next to the server. This may seem obvious. Well, it is obvious.
+Cockpit does not store its own opinion of the state of the server anywhere. The UI always reflect the actual current state of the server and reacts to changes immediately, using the same system APIs that are used by command line tools.
```

Comment:
This is a runon sentence. Lets separate it. ```s/the server and reacts to changes/the server. Cockpit reacts to system changes/```

Reply:
Indeed. Thanks!